client/cucumber/steps/gateway.steps.ts

```diff
@@ -87,8 +92,14 @@ module.exports = function(): any {
         this.contractLanguage = contractLanguage;
     });
 
-    this.When(/^I submit the transaction '(.*?)' with args '(.*?)' ?(and with the transient data '.*?')?$/, this.timeout, async (transaction: string, args: string, transientData: string) => {
-        await this.gatewayHelper.submitTransaction(this.contractName, this.contractVersion, this.contractLanguage, transaction, args, this.gateway, `${this.contractAssetType}Contract`, transientData);
+    this.When(/^I (submit|evaluate) the transaction '(.*?)' with args '(.*?)' ?(?:and with the transient data )?('.*?')?$/, this.timeout, async (submitEvaluate: string, transaction: string, args: string, transientData: string) => {
+        if (submitEvaluate === 'submit') {
```

Comment:
Could you make this a bit tidier (removing the duplicate `this.gatewayHelper.submitTransaction`) by doing something such as:

```
let evaluateBoolean: boolean;
if (submitEvaluate === 'submit'){
    evaluateBoolean = false;
} else {
    evaluateBoolean = true;
}

await this.gatewayHelper.submitTransaction(this.contractName, this.contractVersion, this.contractLanguage, transaction, args, this.gateway, `${this.contractAssetType}Contract`, transientData, evaluateBoolean);

```

Reply:
Done